utils/clients/tests/test_queue_client.py

```diff
@@ -102,16 +103,29 @@ def test_serialise_message(self):
 
     # pylint:disable=no-self-use
     @patch('stomp.connect.StompConnection11.send')
-    def test_send_data(self, mock_stomp_send):
+    def test_send_with_raw_string(self, mock_stomp_send):
         """
-        Test: send establishes a connection and sends given data using stomp.send
-        When: send is called while a valid connection is not held
+        Test: send sends the given data using stomp.send
+        When: send is called with a string argument for message
         """
         client = QueueClient()
-        client.send('dataready', 'test-message')
+        client.send('dataready', 'raw_json_dump')
         (args, _) = mock_stomp_send.call_args
         self.assertEqual(args[0], 'dataready')
-        self.assertEqual(args[1], 'test-message')
+        self.assertEqual(args[1], 'raw_json_dump')
+
+    @patch('stomp.connect.StompConnection11.send')
```

Comment:
same as above re-patch statement

Reply:
It should be a like for like conversion (code shouldn't have to change other than what it is patching

Reply:
Yeah, I see what you mean, I've started testing inner workings of job.Message (which are already covered by test_job).
Will resolve in next commit.